packages/libs/web-common/src/App/Showcase/CardList.jsx

```diff
@@ -3,12 +3,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Select from 'react-select';
 import { Link, RealTimeSearchBox } from '@veupathdb/wdk-client/lib/Components';
-import { projectId } from '@veupathdb/web-common/lib/config';
+import { projectId } from 'ebrc-client/config';
```

Comment:
How does this path get resolved?

Reply:
There is a typescript alias in this package's tsconfig

Reply:
The previous path was causing typescript to try to recompile the compiled code, which caused a "Cannot write file ... because it would overwrite input file." error.